cmd/clusterctl/hack/create-local-repository.py

```diff
@@ -130,6 +131,10 @@ def write_local_repository(provider, version, components_file, components_yaml):
         f.write(components_yaml)
         f.close()
 
+        # Copy CAPI metadata for non-infrastructure providers
+        if not provider.startswith("infrastructure"):
+            copy_file("metadata.yaml", provider_folder)
```

Comment:
This file will now be included in the generated dev-repository for local testing as well.